@Danbaba1/readme.md

```diff
@@ -0,0 +1,6 @@
+# Git & Github training exercises
+Git & Github training exercises for when interacting with a repository that is not yours.
+## How to submit your changes to this repo with Github UI
+Watch this video to learn how to add content to a repo that is not yours, through the Github UI: [YouTube tutorial](https://www.youtube.com/watch?v=2X1FjFqIwIY).
+## Github markdown resources
```

Comment:
Approved! Well done.
Closing this PR because I don't want to merge directly into main.